src/index.js

```diff
@@ -58,21 +57,16 @@ const Users = require('./users');
 const Wappalyzer = require('./wappalyzer');
 const Websocket = require('./websocket');
 
-// base JSON api url
-const BASE = 'http://zap/JSON';
-// base OTHER api url
-const BASE_OTHER = 'http://zap/OTHER';
```

Comment:
Both are needed, they serve different endpoints.

Reply:
What is the use case of BASE_OTHER? As far as I have observed, only BASE was used and I have also catered only BASE, where BASE_OTHER needs to be used?

Reply:
Some endpoints use that URL, e.g.:
https://github.com/zaproxy/zap-api-nodejs/blob/2a2772ef5a1ed72cc533b6e937f80a6c16f920ed/src/exim.js#L100-L104

Reply:
There's an explanation of the format: https://www.zaproxy.org/docs/api/#api-url-format

Reply:
I see, there are other options as well like XML,HTML, etc. Why dont we support them as well and instead of having multiple base URLs for each format, pass the format as parameter to URL?

Reply:
Okay, I am not willing to write a new function just for requesting other endpoint that would be code repetition, I think just having one method that sends the request to all types of endpoints will be a good approach. What do you think about passing another parameter e.g format to `requestNew` method that decides what endpoint to call? Can it be passed from the API calls? The format as parameter thing will help us scale the API to more formats in future and will not be a breaking change for now.

Reply:
It's only code repetition if it does not use the existing method, I'm fine either way though.

Reply:
Done